.github/workflows/java-pr.yml

```diff
@@ -163,7 +164,8 @@ jobs:
           --it-region="us-central1" \
           --it-project="cloud-teleport-testing" \
           --it-artifact-bucket="cloud-teleport-testing-it-gitactions" \
-          --it-private-connectivity="datastream-private-connect-us-central1"
+          --it-private-connectivity="datastream-private-connect-us-central1" \
+          --it-spanner-host="https://staging-wrenchworks.sandbox.googleapis.com/"
```

Comment:
Why is this needed here?
The template just has a parameter which can be used to override the spanner host right?